openquake/hazardlib/nrml_to.py

```diff
@@ -158,15 +158,15 @@ def convert_to(fmt, fnames, chatty=False, *, outdir='.', geometry=''):
                 i += 1
         elif 'nrml/0.4' in root['xmlns']:
             for srcnode in root.sourceModel:
-                appendrow(converter.convert_node(srcnode),
-                          srcs, chatty, sections, s2i)
+                row = converter.convert_node(srcnode)
+                appendrow(row, srcs, chatty, sections, s2i)
         else:  # nrml/0.5
             for srcgroup in root.sourceModel:
                 trt = srcgroup['tectonicRegion']
                 for srcnode in srcgroup:
                     srcnode['tectonicRegion'] = trt
-                    appendrow(converter.convert_node(srcnode),
-                              srcs, chatty, sections, s2i)
+                    row = converter.convert_node(srcnode)
+                    appendrow(row, srcs, chatty, sections, s2i)
```

Comment:
This change and the one above were just for debugging purposes, because this way it becomes a little easier to step into the desired functions via pdb